sdk/src/Core/AWSSDK.Core.WinSub.csproj

```diff
@@ -0,0 +1,280 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
we hand manage this csproj file so this needs to be cleaned up.  Also, since this is for Xamarin, we should use the vs2017 csproj format.

Reply:
Also, how does this dll get packaged?  I don't see nuspec file changes

Reply:
> 
> 
> Also, how does this dll get packaged? I don't see nuspec file changes

The dll will not get published. as this is only referred by test projects